src/main/java/uk/gov/gchq/magmacore/demo/DemoUtils.java

```diff
@@ -16,45 +16,52 @@
 
 import java.util.List;
 
+import uk.gov.gchq.hqdm.rdf.iri.IriBase;
 import uk.gov.gchq.magmacore.service.MagmaCoreService;
+import uk.gov.gchq.magmacore.service.transformation.DbChangeSet;
 import uk.gov.gchq.magmacore.service.transformation.DbTransformation;
 
 /**
- * Functions for creating systems using MagmaCore and HQDM.
+ * Utilities for Magma Core demos.
  */
-public class ExampleDataObjects {
+public class DemoUtils {
+
+    /** IriBase for Reference Data Library. */
+    public static final IriBase REF_BASE = new IriBase("rdl", "http://www.semanticweb.org/magma-core/rdl#");
+
+    /** IriBase for User data. */
+    public static final IriBase USER_BASE = new IriBase("user", "http://www.semanticweb.org/magma-core/user#");
 
     /**
-     * A function that populates a database.
+     * Populate a {@link MagmaCoreService} database.
      *
-     * @param mcService a {@link MagmaCoreService}
-     * @return {@link DbTransformation}
+     * @param mcService {@link MagmaCoreService} to populate.
+     * @return {@link DbTransformation} performed.
      */
     public static DbTransformation populateExampleData(final MagmaCoreService mcService) {
 
-        // Apply the transformation to the database. There are dependencies between these change sets 
-        // since they both depend on RDL being present, but also the occupancies depend on the 
-        // individuals being present, so each change set needs to be applied before the next one 
+        // Apply the transformation to the database. There are dependencies between these change sets
+        // since they both depend on RDL being present, but also the occupancies depend on the
+        // individuals being present, so each change set needs to be applied before the next one
         // can be created.
-        final var rdlChangeSet = ExampleRdl.createRefDataObjects();
+        final DbChangeSet rdlChangeSet = ExampleRdl.createRefDataObjects();
```

Comment:
`var` was introduced into Java to reduce clutter and make the code more readable. It seems you're stuck in old ways!

Reply:
Generally, I agree with this, but in this case I think unless you know the return type of the invoked method (`createRefDataObjects`), then you have to rely on either the name or intellisense to tell you the type. For that reason I tend to prefer explicit type declarations in situations like this to make the code easier to interpret and debug.